_topics/en/pooled-mining.md

```diff
@@ -0,0 +1,293 @@
+documentation for them has never been provided.
+
+[Betterhash][] was a proposed mining pool protocol that allowed pool
+members to select which transactions to include in the blocks they
+create.  The protocol also made it easy for pool members who found a
+new block to submit it directly to the network through their Bitcoin
+full node, which could reduce propagation times and improve the chance
+of that block becoming a permanent part of the block chain.  Additional
+improvements in Betterhash focused on security and efficiently
+distributing work across multiple mining hardware devices.  Although the
+proposal has not been withdrawn, it does appear that development of it
+has been replaced by development on Stratum v2.
+
+[Stratum v2][] is an entirely new protocol developed by several of the
+same people who contributed to Stratum v1.  It provides several of the
+same advantages as Betterhash, although sometimes using different
+mechanisms.  Like Betterhash, one of its key advantages is that it can
+allow individual pool members to choose which transactions to include in
+their blocks.  Stratum v2 is used in production as of this writing,
+although some advanced features are not fully supported by any widely
+deployed mining software.
+
+[BraidPool][] is a proposed design for a fully decentralized mining
+pool.  Unlike a typical centralized mining pool managed with Stratum v1
+or v2, members of a BraidPool must all chose their own transactions and
+funds will be paid for any valid share, preventing the pool from
+interfering with each individual member's choices about what transactions to mine.
+BraidPool is inspired by the former [P2Pool][] decentralized mining pool
+but attempts to mitigate some of P2Pool's problems, including its
+reduced ability to capture fee income due to its large typical
+coinbase transactions and the frequency of P2Pool shares becoming stale.
+BraidPool is under active development as of this writing.
+
+## Pool payout schemes
+
+Previously in this article, we've describe a single idealized payout:
+a miner who creates a 1/100th share for a block template with a block
+reward of 12.3456789 BTC will receive a payout of 0.1234567890 BTC.  In
+practice, no pool we're aware of pays exactly that way, and they use
+slightly different jargon.  The following describes general types of
+payouts, although each pool's implementation of them may differ from our
+descriptions and from other pools.
+
+- **Pay per share (PPS)** pays an amount proportional to each share's
+  PoW.  For example, in a pool that changes a 0% membership fee, each
+  share might be proportional to the block subsidy.  If the subsidy is
+  3.25 BTC and a miner creates a 1/100th share, the miner receives a
+  payment of 0.0325 BTC.  PPS has the advantage of being extremely
+  simple.  A miner can compare all PPS pools and choose the one that
+  advertises the highest rate; the miner can then simply count how many
```

Comment:
Highest rate vs membership fee language above could be confusing (or is for me?)